.prettierignore

```diff
@@ -0,0 +1,3 @@
+.*/**
+docs/
```

Comment:
Prettier can format markdown. Might be useful to set it up to do that in future for files in `docs/`.